edgebuilder-install.sh

```diff
@@ -297,7 +302,7 @@ install_node()
     echo "WARN: docker.socket is enabled, disabling..."
     systemctl disable docker.socket
   fi
-  for i in docker docker-engine docker.io containerd runc; do
+  for i in docker docker-engine docker.io containerd runc docker-ce docker-ce-cli docker-compose-plugin; do
```

Comment:
Could you please add "docker-ce-rootless-extras" to this list? Thanks

Reply:
When I tested the installer I didn't see that this was installed. But that's fine I've added it